railties/test/generators/app_generator_test.rb

```diff
@@ -822,7 +844,7 @@ def test_no_skip_javascript_option_with_no_skip_javascript_argument
   end
 
   def test_hotwire
-    run_generator_and_bundler [destination_root]
+    run_generator_and_bundler [destination_root, "--no-skip-hotwire"]
```

Comment:
But adding --no-skip-hotwire to this argument, we can specifically test for hotwire.

Reply:
`--no-skip-hotwire` is the default behavior — no option is necessary.  That is what this test is testing.